runtime/browser/cdn/browser-cdn-loader.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright (c) 2017 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+"use strict";
+
+const Loader = require('../../loader.js');
+const particle = require("../../particle.js");
+const DomParticle = require("../../dom-particle.js");
+
+module.exports = class BrowserLoader extends Loader {
+  constructor(urlMap) {
```

Comment:
The path is generally relative to ArcsLib.js, but not always. Also the entity that loads ArcsLib can know that those paths relate, but because at a primitive level I'm not defining how that artifact is loaded so there is no general solution. The `playground` for example, knows the paths to be the same and configures the `urlMap` using that knowledge.

Yes, `worker-entry-cdn.js` was just a way for me to implement what I needed without disturbing the existing (non-cdn) codebase. It certainly can go away, but I didn't want to block all the CDN stuff on working out the details of the loader.